protocol/communities/manager.go

```diff
@@ -4701,6 +4701,13 @@ func (m *Manager) AddCommunityToken(token *community_token.CommunityToken) (*Com
 		if err != nil {
 			return nil, err
 		}
+
+		if token.PrivilegesLevel == community_token.OwnerLevel {
+			community, err = m.updatePrivateKeyAndControlNode(community, m.identity)
```

Comment:
FYI, @osmaczko updated this method, so you'll have a conflict (whoever merges first)

Reply:
The method is named `PromoteSelfToControlNode` now, there is no need to pass `m.identity`.